Lib/ufo2ft/outlineCompiler.py

```diff
@@ -410,6 +410,134 @@ def setupTable_cmap(self):
+                codepageRanges.add(20)          # Traditional chinese
+            elif code == 0xACF4:
+                codepageRanges.add(21)          # Korean Johab
+            elif code == 0x2030 and hasAscii:
+                hasMac = True
+            elif code == 0x2665 and hasAscii:
+                codepageRanges.add(30)          # OEM
+            elif 0xF000 <= code <= 0xF0FF:
+                codepageRanges.add(31)          # Symbol
+            elif code == 0x255A and hasAscii:
+                codepageRanges.add(62)          # WE/Latin 1
+                codepageRanges.add(63)          # US
+            elif code == 0x00C5 and hasAscii:
+                hasCP865 = True
+            elif code == 0x00E9 and hasAscii:
+                hasCP863 = True
+            elif code == 0x00F5 and hasAscii:
+                hasCP860 = True
+            elif code == 0x00FE and hasAscii:
+                hasCP861 = True
+            elif code == 0x2524:
+                hasLineart
+            elif code == 0x0405:
+                hasCP855 += 1
+            elif code == 0x255C:
+                hasCP866 += 1
+            elif code == 0x00BD:
+                hasCP869 += 1
+            elif code == 0x221A:
+                hasRadical = True
+            elif code == 0x2211:
+                hasSummation = True
+
+        if hasMac and hasSummation:
+            codepageRanges.add(29)              # Mac Roman
+        if hasCP869 == 2 and hasLineart:
+            codepageRanges.add(48)              # IBM Greek
+        if hasCP866 == 2 and hasLineart:
+            codepageRanges.add(49)              # MS DOS Russian
+        if hasCP865 and hasLineart and hasRadical:
+            codepageRanges.add(50)              # MS-DOS Nordic
+        if hasCP864 and hasRadical:
+            codepageRanges.add(51)              # Arabic
+        if hasCP863 and hasLineart and hasRadical:
+            codepageRanges.add(52)              # MS-DOS Canadian French
+        if hasCP862 and hasLineart and hasRadical:
+            codepageRanges.add(53)              # Hebrew
+        if hasCP861 and hasLineart:
+            codepageRanges.add(54)              # MS-DOS Icelandic
+        if hasCP860 and hasLineart and hasRadical:
+            codepageRanges.add(55)              # MS-DOS Portuguese
+        if hasCP857 and hasLineart:
+            codepageRanges.add(56)              # IBM turkish
+        if hasCP855 == 2 and hasLineart:
+            codepageRanges.add(57)              # IBM Cyrillic
+        if hasCP852 and hasLineart:
+            codepageRanges.add(58)              # Latin2
+        if hasCP775 and hasLineart:
+            codepageRanges.add(59)              # Baltic
+        if hasCP737 and hasLineart and hasRadical:
+            codepageRanges.add(60)              # Greek, former 437 G
+        if hasCP708 and hasLineart:
+            codepageRanges.add(61)              # Arabic; ASMO 708
+
+        return list(codepageRanges)
```

Comment:
it can stay a set. the `intListToNum` only checks if some integer is contained in that sequence